stage1/init/common/mount.go

```diff
@@ -66,44 +92,84 @@ func GenerateMounts(ra *schema.RuntimeApp, volumes map[types.ACName]types.Volume
 			defaultMode := "0755"
 			defaultUID := 0
 			defaultGID := 0
+			uniqName := ra.Name + "-" + mp.Name
 			emptyVol := types.Volume{
-				Name: mp.Name,
+				Name: uniqName,
 				Kind: "empty",
 				Mode: &defaultMode,
 				UID:  &defaultUID,
 				GID:  &defaultGID,
 			}
 
+			dockerImplicit := convertedFromDocker(ra)
+
 			log.Printf("warning: no volume specified for mount point %q, implicitly creating an \"empty\" volume. This volume will be removed when the pod is garbage-collected.", mp.Name)
```

Comment:
Is this message still correct? If `dockerImplicit` is `true`, then it won't be "empty" anymore, right?